pkg/sync/v15/data/ClusterRole.rbac.authorization.k8s.io/admin.yaml

```diff
@@ -19,6 +19,24 @@ rules:
   - access
   verbs:
   - create
+- apiGroups:
```

Comment:
same. Do we need this if we use aggregates?

Reply:
same :) 
as above
this is exactly my question. 
without this sync pod is continuously updating, because there are ALWAYS changes from sync pod perspective.  
https://github.com/openshift/openshift-azure/pull/2129#issuecomment-573478271
https://github.com/openshift/openshift-azure/pull/2129#issuecomment-573901132
@mjudeikis